src/interfaces/exchange.ts

```diff
@@ -3,10 +3,17 @@ import Observable from 'zen-observable-ts';
 import { CombinedError } from '../modules/error';
 import { IOperation } from './operation';
 
+type Scalar = string | number | boolean;
+
+export interface IExecutionData {
+  __typename?: string;
```

Comment:
This seems to cause issues as other types in the library aren’t adapted for it.

This type should be fine as long as we can let the client component retune the result (separate PR?)

But let’s not couple the entire exchange’s type to “__typename” as another cache exchange or otherwise might not be relying on this :) e.g. a schema based cache